### PR TITLE
Added image driver global param file docs

### PR DESCRIPTION
**Pull request by yixinmao**
- Added image driver global parameter file docs
- Added/fixed a few global parameters in the docs for classic driver

**NOTE**: description for some variables are still missing or ambiguous! A list of such variables whose description to be added/fixed:
  - SHARE_LAYER_MOIST
  - OUT_TIME_UNITS
  - BLOWING_VAR_THRESHOLD
  - BLOWING_CALC_PROB
  - BLOWING_SIMPLE
  - BLOWING_FETCH
  - BLOWING_SPATIAL_WIND
  - ALB_SRC

### CODE REVIEW

**Comment by jhamman** [comment]
`OUT_TIME_UNITS` description: Units for output time variables.  Valid options are SECONDS, MINUTES, HOURS, or DAYS.

**Comment by tbohn** [comment]
Re: `SHARE_LAYER_MOIST`: When `TRUE`, then *if* the soil moisture in the layer that contains more than half of the roots is above the critical point, then the plant's roots in the drier layers can access the moisture of the wetter layer so that the plant does not experience moisture limitation.  If `SHARE_LAYER_MOIST` is `FALSE` *or* all of the soil layer moistures are below the critical point, transpiration in each layer is limited by the layer's soil moisture.  `SHARE_LAYER_MOIST` is `TRUE` by default (this has been the default model behavior since release 4.0.3 at least).

Sorry if that's too long of an explanation.  But you get the idea.

File: docs/Documentation/Drivers/Classic/GlobalParam.md [hunk 1]
```diff
@@ -197,25 +208,25 @@ _Examples._ a standard four column daily forcing data file will be defined as:
...
+| ALBEDO_SRC         | string          | N/A                 | This option tells VIC where to look for ALBEDO values:FROM_VEGLIB = Use the ALBEDO values listed in the vegetation library file.FROM_VEGPARAM = Use the ALBEDO values listed in the vegetation parameter file. Note: for this to work, VEGPARAM_ALB must be TRUE.FROM_VEGHIST = Use the ALBEDO values listed in the veg_hist forcing files. Note: for this to work, ALBEDO must be supplied in the veg_hist files and listd in the global parameter file as one of the variables in the files. Default = FROM_VEGLIB.                                                                                                                                                     |
```
Comment (bartnijssen) [inline docs/Documentation/Drivers/Classic/GlobalParam.md]: There is no `ALBEDO_SRC` in the code. The description should be re-assigned to `ALB_SRC` and the `ALBEDO_SRC` entry can be deleted. Also requires that the following two files are updated:

./tests/science/global_param.classic.4.2_ecflux.txt:# ALBEDO_SRC      FROM_VEGPARAM
./tests/science/global_param.classic.5.0_ecflux.txt:# ALBEDO_SRC            FROM_VEGPARAM

**Comment by bartnijssen** [comment]
Here are some of the others:

`BLOWING_VAR_THRESHOLD` | boolean | N/A | When this option is set to TRUE a variable shear stress threshold is used to determine the blowing snow flux. If FALSE then a fixed threshold is used. See Li and Pomeroy (1997) for details. Default is TRUE.

 `BLOWING_CALC_PROB` | boolean | N/A | When this option is set to TRUE, the probability of occurrence of blowing snow is calculated as a function of environmental conditions. If FALSE then the probability is set to 1. See Lu and Pomeroy (1997) for details. Default is TRUE.

 `BLOWING_SIMPLE` | boolean | N/A | When this option is set to TRUE, the sublimation flux of blowing snow is calculated as a function vapor pressure and wind speed. If FALSE then additional calculations are made to account for a saltation and suspension layer. See Lu and Pomeroy (1997) for details. Default is FALSE.

 `BLOWING_FETCH` | boolean | N/A | This option is only used when `BLOWING_SIMPLE` is set to FALSE. When this option is set to TRUE, the fetch is accounted for in the calculation of the sublimation flux from blowing snow. If FALSE then the fetch is not used. See Lu and Pomeroy (1997) for details. Default is TRUE.

 `BLOWING_SPATIAL_WIND` | boolean | N/A | When this option is set to TRUE, multiple wind speed ranges, calculated according to a probability distribution, are used to determine the sublimation flux from blowing snow. If FALSE then a single wind speed is used. See Lu and Pomeroy (1997) for details. Default is TRUE.

**Comment by bartnijssen** [comment]
@yixinmao : I think this is all the info you need to complete the PR

**Comment by yixinmao** [comment]
@jhamman @bartnijssen All comments incorporated - ready to merge after passing the tests.

**Comment by bartnijssen** [comment]
@yixinmao : Can you update with the base branch

**Comment by yixinmao** [comment]
@bartnijssen Done - merged develop branch.

**Comment by jhamman** [comment]
closes #599

**Pull request merged**